src/sonic-config-engine/config_samples.py

```diff
@@ -47,6 +47,10 @@ def generate_l2_config(data):
     if not data['DEVICE_METADATA']['localhost'].has_key('type'):
         data['DEVICE_METADATA']['localhost']['type'] = 'ToRRouter'
     data['VLAN'] = {'Vlan1000': {'vlanid': '1000'}}
+    vp = [] 
+    for port in natsorted(data['PORT'].keys()):
+        vp.append(port)
```

Comment:
Is it same as:
vp = natsorted(data['PORT'].keys())
?

Reply:
Agree with comments. I will update the review with code changes.